README.md

```diff
@@ -1,18 +1,20 @@
-<p align="center">
+<div align="center">
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
```

Comment:
needs `#contributors` instead of `#contributors-`, otherwise the link doesn't work